### PR TITLE
Feature: redesign project grid

### LINKED ISSUE #252: HomePage: redesign project grid
## Description

Redesign the project grid:
- Creation button
- 2x6 project cards
- global spacing

Not to implements:
- tags (local, remote, ...)
- search bar
- project-card

Expected design:

image

### PR DESCRIPTION
**Pull request by Zorin95670**
Resolved #252

### CODE REVIEW

File: src/components/grid/ProjectGrid.vue [hunk 1]
```diff
@@ -1,47 +1,68 @@
...
+          class="row q-mb-lg justify-center items-center col-2"
+        >
+          <project-card
+            :project="project"
+          />
+        </div>
       </div>
     </div>
+    <div
+      v-if="Object.keys(projects).length === 0"
```
Comment (justinehell) [inline src/components/grid/ProjectGrid.vue:60]: Only `projects.length === 0` since it's now an Array

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/components/grid/ProjectGrid.vue (hunk 1 above)
Comment (Zorin95670) [reply src/components/grid/ProjectGrid.vue]: done

File: src/pages/HomePage.vue [hunk 2]
```diff
@@ -82,3 +82,9 @@ onUnmounted(() => {
   updateProjectSubscription.unsubscribe();
 });
 </script>
+
+<style lang="scss" scoped>
+.project-grid {
```
Comment (justinehell) [inline src/pages/HomePage.vue:87]: new class but I don't see it in template

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/components/grid/ProjectGrid.vue [hunk 3]
```diff
@@ -1,47 +1,68 @@
...
+    <div class="row no-wrap">
+      <div class="row">
         <q-btn
           outline
           no-caps
-          class="q-ml-xl"
           color="primary"
-          icon="fa-solid fa-plus"
-          :label="$t('actions.home.newProject')"
+          class="square-button"
```
Comment (k-y0u) [inline src/components/grid/ProjectGrid.vue:29]: could be more explicit like "create-project-button" or "new-project-button"

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/grid/ProjectGrid.vue (hunk 3 above)
Comment (Zorin95670) [reply src/components/grid/ProjectGrid.vue]: done :heart:

**Review by justinehell — APPROVED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**